### ISSUE TITLE
[BUG] Image not loading / displaying in nested adaptive card

### ISSUE THREAD

**Issue by joschi27** [MicrosoftDocs/msteams-docs#3883]
Good day,

**TLDR;** When sending an adaptive card with another inside that can be opened with Action.ShowCard the inner card does not display any images.

**The issue**
I'm trying to send an adaptive card with expandable content over the bot framework and wanted to use the showcard action for this. I discovered that the image does not render (and at the start it was because the image was too big). Afterwards, I believed that because Images get proxied over urlp.asm.skype.com, my webserver was serving the images wrong (or wrong mimetype or whatever). This was not the case as other images displayed correctly. After testing, I found out that all images inside of the nested adaptive card will not display.

**How to reproduce:**
Send this JSON as an adaptive card to a teams channel:

`{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "Image",
      "url": "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Microsoft_Office_Teams_%282018%E2%80%93present%29.svg/258px-Microsoft_Office_Teams_%282018%E2%80%93present%29.svg.png",
      "size": "Medium"
    }
  ],
  "actions": [
    {
      "type": "Action.ShowCard",
      "title": "Action.ShowCard",
      "card": {
        "type": "AdaptiveCard",
        "body": [
          {
            "type": "Image",
            "url": "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Microsoft_Office_Teams_%282018%E2%80%93present%29.svg/258px-Microsoft_Office_Teams_%282018%E2%80%93present%29.svg.png",
            "size": "Medium"
          }
        ]
      }
    }
  ]
}`

The image does not get rendered regardless of _web_ or _desktop_ teams application. (I assume it's because the desktop teams-app just renders a webpage)

-> The inner nested adaptive card does not have the _src_ attribute, only the _ng-src_ attribute. If you add the src attribute, it displays correctly. So it might just be an angularjs thing.

Also, your webclient throws a LOT of errors in the console, some angular, some not. It's hard to find the errors that are actually caused by my nested card.

**Comment by Prithvi-MSFT** [comment]
Hi,
We were able to repro this issue. The card works fine in adaptive card designer but does not work in teams. This seems like a platform bug. We have raised a request with concerned team.

**Comment by v-chetsh** [comment]
@joschi27 - Sorry for the late response.
This issue has already resolved, Please check and confirm here.
